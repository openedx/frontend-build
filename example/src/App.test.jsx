/* eslint-disable global-require */
import renderer from 'react-test-renderer';
import App from './App';

describe('Basic test', () => {
  it('should render', () => {
    const Component = <App />;
    const tree = renderer.create(Component);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
