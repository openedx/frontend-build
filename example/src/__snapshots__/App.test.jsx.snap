// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic test should render 1`] = `
<div>
  <h1>
    Test page
  </h1>
  <h2>
    SCSS parsing tests
  </h2>
  <h3>
    The Apples
  </h3>
   ("The Apples" should be red)
  <h2>
    ES6 parsing tests
  </h2>
  <ul>
    <li>
      macintosh
    </li>
    <li>
      granny smith
    </li>
    <li>
      fuji
    </li>
    <li>
      golden delicious
    </li>
  </ul>
  <h2>
    JSX parsing tests
  </h2>
  <IconMock
    style={
      Object {
        "width": "10rem",
      }
    }
  />
  <h2>
    Asset import tests
  </h2>
  <img
    alt="apple"
    src="icon/mock/path"
    style={
      Object {
        "width": "10rem",
      }
    }
  />
  <img
    alt="apple"
    src="icon/mock/path"
    style={
      Object {
        "width": "10rem",
      }
    }
  />
  <br />
  <img
    alt="apple"
    src="test-file-stub"
    style={
      Object {
        "width": "10rem",
      }
    }
  />
  <p>
    Photo by Louis Hansel @shotsoflouis on Unsplash
  </p>
  <h2>
    process.env tests
  </h2>
  <p>
    Test process.env variable: 
    foo
  </p>
  <p>
    Non-existent process.env variable (nothing here is good): 
  </p>
  <h2>
    env.config.js tests
  </h2>
  <p>
    <span>
      env.config.js boolean test: 
    </span>
    Good, false meant false.  We did not cast a boolean to a string.
  </p>
  <p>
    env.config.js integer test: 
    It was an integer. Great.
  </p>
  <h2>
    Right-to-left language handling tests
  </h2>
  <p
    className="text-align-right"
  >
    I'm aligned right, but left in RTL.
  </p>
</div>
`;
